src/Task/Assets/ServiceProvider.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Robo\Task\Assets;
+
+use League\Container\ServiceProvider\AbstractServiceProvider;
+
+class ServiceProvider extends AbstractServiceProvider
+{
+    protected $provides = [
+        'taskMinify',
+        'taskImageMinify',
+        'taskLess',
+        'taskScss',
+    ];
+
+    public function register()
+    {
+        $this->getContainer()->add('taskMinify', Minify::class);
+        $this->getContainer()->add('taskImageMinify', ImageMinify::class);
+        $this->getContainer()->add('taskLess', Less::class);
+        $this->getContainer()->add('taskScss', Scss::class);
+    }
```

Comment:
We also need to check to see if $service is null, and return array_keys($tasks) in this instance.